locales/fr.json

```diff
@@ -401,26 +402,7 @@
   "Please share your feedback.": "Please share your feedback.",
   "You may also be able to get help at a free tax clinic. ": "You may also be able to get help at a free tax clinic. ",
   " if you can use a tax clinic.": " if you can use a tax clinic.",
-  "Small and rural communities": "Small and rural communities",
   "Did you live in a small and rural community in 2018?": "Did you live in a small and rural community in 2018?",
-  "Check if you live in a small and rural community": "Check if you live in a small and rural community",
-  "You live in a small and rural community if you live outside of:": "You live in a small and rural community if you live outside of:",
-  "Barrie": "Barrie",
-  "Belleville": "Belleville",
-  "Brantford": "Brantford",
-  "Greater Toronto Area": "Greater Toronto Area",
-  "Guelph": "Guelph",
-  "Hamilton": "Hamilton",
-  "Kingston": "Kingston",
-  "Kitchener-Cambridge-Waterloo": "Kitchener-Cambridge-Waterloo",
-  "London": "London",
-  "Oshawa": "Oshawa",
-  "Ottawa": "Ottawa",
-  "Peterborough": "Peterborough",
-  "St. Catharines-Niagara": "St. Catharines-Niagara",
-  "Sudbury": "Sudbury",
-  "Thunder Bay": "Thunder Bay",
-  "Windsor": "Windsor",
```

Comment:
Are these actually deleted or no? I thought we had a big list now (on the Climate Action incentive page)